docs/xgcpolicy.md

```diff
@@ -136,15 +136,16 @@ The following options are ignored when specified with `-Xgcpolicy:balanced`:
 
 : The metronome policy is an incremental, deterministic garbage collector with short pause times. Applications that are dependent on precise response times can take advantage of this technology by avoiding potentially long delays from garbage collection activity. The metronome policy is supported on specific hardware and operating system configurations.
 
-    For more information, see [Using the Metronome Garbage Collector](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mm_gc_mgc.html).
+    For more information about this policy, see [Garbage collection: 'metronome' policy](gc.md#metronome-policy).
+<!-- [Using the Metronome Garbage Collector](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mm_gc_mgc.html). -->
 
 #### Defaults and options
 
 `-Xgc:synchronousGCOnOOM | -Xgc:nosynchronousGCOnOOM`
 : One occasion when garbage collection occurs is when the heap runs out of memory. If there is no more free space in the heap, using `-Xgc:synchronousGCOnOOM` stops your application while garbage collection removes unused objects. If free space runs out again, consider decreasing the target utilization to allow garbage collection more time to complete. Setting `-Xgc:nosynchronousGCOnOOM` implies that when heap memory is full your application stops and issues an out-of-memory message. The default is `-Xgc:synchronousGCOnOOM`.
 
 `-Xnoclassgc`
-: Disables class garbage collection. This option switches off garbage collection of storage associated with Java classes that are no longer being used by the OpenJ9 VM. The default behavior is -Xnoclassgc.
+: Disables class garbage collection. This option switches off garbage collection of storage associated with Java classes that are no longer being used by the OpenJ9 VM. The default behavior is `-Xnoclassgc`.
```

Comment:
default is -Xclassgc (not -Xnoclassgc), which heuristically decides which GC cycle will attempt to unload classes.

We also have -Xalwaysclassgc that will attempt to unload classes on each applicable GC (what is Global GC).

These 3 options are not applicable to Balanced  GC polices. Class unloading there is done incrementally in Partial GCs. There are a couple of -XXgc: options that can disable/enable it, but I don't know if we want to expose them (we generally don't document -XXgc options)

Reply:
Default changed.
-Xalwaysclassgc added.
3 options removed from Balanced GC section.